### DISCUSSION TITLE
multinma request

### DISCUSSION THREAD

**Discussion by salvo9g** [dmphillippo/multinma#14] [General]
hello, I am a researcher in the medical and psychological fields. I'm looking for a way to do moderation analysis on NMA. I wonder if multinma is a pacjage that allows you to do this kind of analysis? (I don't have individual data but only aggregates)

**Comment by dmphillippo** [comment]
Hi @salvo9g, you can fit meta-regression models using the multinma package. The `regression` argument to the [`nma()`](https://dmphillippo.github.io/multinma/reference/nma.html) function lets you specify a regression model, e.g. `~age:.trt` will include the interaction of age and treatment. 

There are a few example vignettes which demonstrate this: [atrial fibrillation](https://dmphillippo.github.io/multinma/articles/example_atrial_fibrillation.html), [statins](https://dmphillippo.github.io/multinma/articles/example_statins.html), and [BCG vaccine](https://dmphillippo.github.io/multinma/articles/example_bcg_vaccine.html).

**Comment by salvo9g** [comment]
Hi,

thanks for your kind reply.

Salvo

Il giorno lun 21 feb 2022 alle ore 10:39 David Phillippo <
***@***.***> ha scritto:
